usl_pipeline/cloud_functions/main.py

```diff
@@ -582,8 +582,22 @@ def _build_feature_matrix_from_archive(
             else:
                 logging.warning(f"Unexpected member name: {name}")
 
-    if any((elevation, buildings, green_areas, soil_classes)):
-        if not all((elevation, buildings, green_areas, soil_classes)):
+    if any(
```

Comment:
this might be clearer written out as
```python
files_present = [file is not None for file in (elevation, buildings, green_areas, soil_classes)]
if any(files_present):
  if not all(files_present):
```

Reply:
Yep, good idea!